styles.css

```diff
@@ -1241,7 +1240,14 @@ button.project-button:active {
   }
 
   .popups-project-p-desktop {
-    margin-top: 6%;
+    margin-top: 7%;
+    margin-left: 4%;
+    font-family: 'Poppins', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 30px;
+    color: #344563;
   }
```

Comment:
You did a great job fixing the mobile popup 😄:
![image](https://user-images.githubusercontent.com/42799579/211073383-bea2aa1a-2f1e-4c1f-8985-c9ea8efe05b2.png)

Please keep in mind that the desktop one still needs to be fixed, as requested by the previous reviewer 😉. Kindly make sure to adjust all the elements inside the container:
![image](https://user-images.githubusercontent.com/42799579/211073434-0bf41f9b-79a6-4515-a9f8-d57e332dc124.png)

Reply:
Hi Gabriel. 
I don't know how you look at it but that doesn't make sense. When I check the project according to Figma dimensions it looks ok for both mobile and desktop. You can see the screenshots below.
![Screen Shot 2023-01-06 at 22 20 44](https://user-images.githubusercontent.com/44959066/211084880-3734db3a-2e03-4812-bc46-bee6123789aa.png)
![Screen Shot 2023-01-06 at 22 23 32](https://user-images.githubusercontent.com/44959066/211084952-f4f3bab3-bcaf-4da6-873b-943e927f552f.png)